libraries/AP_Logger/AP_Logger_MAVLink.h

```diff
@@ -49,7 +49,7 @@ class AP_Logger_MAVLink : public AP_Logger_Backend
     // erase handling
     void EraseAll() override {}
 
-    bool NeedPrep() override { return false; }
+    void PrepForArming() override {}
```

Comment:
This makes me a little nervous as things may get added to PrepForArming over time.  Perhaps factoring something out of `PrepForArming` would help there.

Reply:
If I factor it out I have to duplicate it. Mavlink is the odd man out here.